docs/content/guides/dagster/branch-depl-with-snowflake.mdx

```diff
@@ -0,0 +1,348 @@
+import requests
+
+from dagster import asset
+
+
+@asset(
+    config_schema={"N": int},
+    io_manager_key="snowflake_io_manager",
+)
+def items(context) -> pd.DataFrame:
+    """Items from the Hacker News API: each is a story or a comment on a story."""
+    rows = []
+    max_id = requests.get(
+        "https://hacker-news.firebaseio.com/v0/maxitem.json", timeout=5
+    ).json()
+    # Hacker News API is 1-indexed, so adjust range by 1
+    for item_id in range(max_id - context.op_config["N"] + 1, max_id + 1):
+        item_url = f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        rows.append(requests.get(item_url, timeout=5).json())
+
+    # ITEM_FIELD_NAMES is a list of the column names in the Hacker News dataset
+    result = pd.DataFrame(rows, columns=ITEM_FIELD_NAMES).drop_duplicates(subset=["id"])
+    result.rename(columns={"by": "user_id"}, inplace=True)
+    return result
+
+
+@asset(
+    io_manager_key="snowflake_io_manager",
+)
+def comments(items: pd.DataFrame) -> pd.DataFrame:
+    """Comments from the Hacker News API."""
+    return items[items["type"] == "comment"]
+
+
+@asset(
+    io_manager_key="snowflake_io_manager",
+)
+def stories(items: pd.DataFrame) -> pd.DataFrame:
+    """Stories from the Hacker News API."""
+    return items[items["type"] == "story"]
+```
+
+As you can see, our assets use a [I/O manager](/concepts/io-management/io-managers) named `snowflake_io_manager` (source code available in the [Transitioning Data Pipelines from Development to Production](/guides/dagster/local-to-production-tutorial#part-one-local-development) guide). Using I/O managers and other resources allow us to swap out implementations per environment without modifying our business logic.
+
+---
+
+## Step 2: Configure our job for each environment
+
+At runtime, we’d like to determine which environment our code is running in: branch deployment, or production. This information dictates how our code should execute, specifically with which credentials and with which database.
+
+To ensure we can't accidentally write to production from within our branch deployment, we’ll use a different set of credentials from production and write to our database clone.
+
+Dagster automatically sets certain environment variables containing deployment metadata, allowing us to read these environment variables to discern between deployments. We can access the `DAGSTER_CLOUD_IS_BRANCH_DEPLOYMENT` environment variable to determine the currently executing environment. Refer to [TODO: PAGE TITLE]() for a full list of preset environment variables.
+
+Because we want to configure our assets to write to Snowflake using a different set of credentials and database in each environment, we’ll configure a separate I/O manager for each environment:
+
+```python file=/guides/dagster/local_to_production/branch_depl_tutorial/repository_v1.py startafter=start_repository endbefore=end_repository
+@repository
+def repo():
+    snowflake_config = {
```

Comment:
This is what the config ends up looking like for production in the local to prod guide 

```
{
                    "account": "abc1234.us-east-1",
                    "user": "system@company.com",
                    "password": {"env": "SYSTEM_SNOWFLAKE_PASSWORD"},
                    "database": "PRODUCTION",
                    "schema": "HACKER_NEWS",
                }
```

i think making `account` `user` and `password` match between the guides would help with continuity for someone who is using both to set up prod and branch deployments